custom_components/fyta/manifest.json

```diff
@@ -10,9 +10,9 @@
   "homekit": {},
   "iot_class": "cloud_polling",
   "requirements": [
-    "fyta_cli==0.2.3"
+    "fyta_cli==0.3.0"
   ],
   "ssdp": [],
-  "version": "0.1.0",
+  "version": "0.3.1",
```

Comment:
Does this have to be in line with the version of fyta_cli? While same version numbers in the beginning might make sense, I would assume that the custom component might have more releases than fyta_cli in the future. Therefore I think we can keep a separate version tree. What do you think?